js/packages/core/index.ts

```diff
@@ -152,6 +152,9 @@ class Renderer {
   private _sendMessage: Function;
   private _nextRefId: number;
 
+  // Fades applied to roots when they are activated and deactivated as a result of render
+  rootFades: { inMs: number, outMs: number} = { inMs: 20, outMs: 20 };
```

Comment:
Are these public with the intention of a user writing property updates after constructing the renderer if they want to customize fade times?

```
let x = new Renderer((batch) => something(batch));
x.rootFades.inMs = 8;
x.rootFades.outMs = 8;
x.render(...);
```

Personally I'm not a big fan of the imperative style there; I'd prefer that we either:
* Make these things constant for the lifetime of the Renderer instance, in which case we take these as constructor args that you don't get to change later
* Make these arguments to the render() function so that it's clear (and more declarative) with each invocation what your fade expectations are

Reply:
ha, I thought you might not like this, but I agree it does feel a little gross. I think then I'll go with adding a separate `renderWithOptions` so we don't break the existing interface. sound good?